x/participation/module_simulation.go

```diff
@@ -18,6 +18,10 @@ const (
 	// TODO: Determine the simulation weight value
 	defaultWeightMsgParticipate int = 100
 
+	opWeightMsgWithdrawAllocations = "op_weight_msg_create_chain"
```

Comment:
```suggestion
	opWeightMsgWithdrawAllocations = "op_weight_withdraw_allocations"
```

Reply:
We merged the fix on Starport, so if you scaffold a new message in the future, you can pull the latest changes on Starport @giunatale